uptime_service_validation/coordinator/helper.py

```diff
@@ -42,7 +42,7 @@ def getPreviousStatehash(conn, logger, bot_log_id):
     cursor = conn.cursor()
     try:
         sql_query = """select  ps.value parent_statehash, s1.value statehash, b.weight
-            from botlogs_statehash b join statehash s1 ON s1.id = b.statehash_id 
+            from bot_logs_statehash b join statehash s1 ON s1.id = b.statehash_id 
```

Comment:
I think we also need corresponding change in createDB.sql script (https://github.com/MinaFoundation/uptime-service-validation/blob/main/uptime_service_validation/database/createDB.sql)

Reply:
Right. Done.